charts/immich/Chart.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v2
 description: A chart to power Immich (immich.app) running on kubernetes
 name: immich
-version: 0.6.0
+version: 0.6.1
```

Comment:
This shouldn't need a version bump

Reply:
chart immutability will break no ? we might need to change that section as well in CONTRIBUTING.md.

Reply:
If the version doesn't change then the chart release doesn't run at all